pom.xml

```diff
@@ -596,6 +597,65 @@
                     <argLine>-Xmx1024m</argLine>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.rat</groupId>
+                <artifactId>apache-rat-plugin</artifactId>
+                <version>${apache.rat.version}</version>
+                <configuration>
+                    <licenses>
+                        <license>
+                            <family>AL</family>
+                            <text>Licensed under the Apache License, Version 2.0 (the
+                                "License")</text>
+                            <text>Licensed to the Apache Software Foundation (ASF) under
+                                one
+                                or more contributor license agreements; and to You under
+                                the
+                                Apache License, Version 2.0.</text>
+                            <text>http://www.apache.org/licenses/LICENSE-2.0</text>
+                            <text>https://www.apache.org/licenses/LICENSE-2.0</text>
+                            <text>http://www.apache.org/licenses/LICENSE-2.0.html</text>
+                            <text>https://www.apache.org/licenses/LICENSE-2.0.html</text>
+                            <text>http://www.apache.org/licenses/LICENSE-2.0.txt</text>
+                            <text>https://www.apache.org/licenses/LICENSE-2.0.txt</text>
+                        </license>
+                    </licenses>
+                    <families>
+                        <family>
+                            <id>AL</id>
+                            <name>Apache License Version 2.0</name>
+                        </family>
+                    </families>
+                    <excludes>
+                        <exclude>NOTICE</exclude>
+                        <exclude>DISCLAIMER</exclude>
+                        <exclude>**/.*</exclude>
+                        <exclude>**/*.json</exclude>
+                        <exclude>**/*.log</exclude>
+                        <exclude>**/*.png</exclude>
+                        <exclude>**/*.jpg</exclude>
+                        <exclude>**/*.avsc</exclude>
```

Comment:
I think we need to include avro schemas. They are included in other the Hudi repo